core/src/main/java/lucee/runtime/functions/conversion/SerializeJSON.java

```diff
@@ -84,7 +84,7 @@ else if (!StringUtil.isEmpty(useSecureJSONPrefixOrCharset)) {
 
 	private static String _call(PageContext pc, Object var, Object queryFormat, Charset charset, boolean useSecureJSONPrefix) throws PageException {
 		try {
-			JSONConverter json = new JSONConverter(true, charset, JSONDateFormat.PATTERN_CF);
+			JSONConverter json = new JSONConverter(false, charset, JSONDateFormat.PATTERN_CF);
```

Comment:
this jumps out to be as being wrong?

Reply:
Hello,

JSONConverter constructor's first argument is a boolean flag for ignoreRemotingFetch. It basically determines if the converter should ignore the remotingFetch attribute on a property. In JSONConverter.java, this flag is referenced when the converter is converting a component, which is where the remotingFetch attribute would be used on a property.

Since the goal was to make the SerializeJSON function not ignore that flag, I am passing 'false' for the first argument there.

Thank you.

Reply:
@zspitzer I committed & pushed additional test cases to cover different behaviors of the SerializeJSON function.

Thank you.